src/main/java/Duke.java

```diff
@@ -1,10 +1,215 @@
+import java.util.*;
 public class Duke {
+
+    public static String[] processInput(String in) {
+        String[] strings = new String[2];
+        int index = in.indexOf(" ");
+        String subString1 = in.substring(index+1);
+        if (!subString1.contains("/")) {
+            strings[0] = subString1;
+            return strings;
+        }
+        else {
+            int index1 = subString1.indexOf("/");
+            String subString2 = subString1.substring(0, index1-1);
+            String subString3 = subString1.substring(index1);
+            int index2 = subString3.indexOf(" ");
+            String subString4 = subString3.substring(index2+1);
+            strings[0] = subString2;
+            strings[1] = subString4;
+            return strings;
+        }
+    }
+
+    public static int listLength(Task[] tasks) {
+        int length = 0;
+        for (int i = 0; i<tasks.length; i++) {
+            if (tasks[i] != null) {
+                length++;
+            }
+        }
+        return length;
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
```

Comment:
I think that maybe the main function can be more logically structured.
if it is possible, varying the level of abstraction within a code fragment should be avoided.